airbyte-integrations/connectors/destination-clickhouse/src/main/kotlin/io/airbyte/integrations/destination/clickhouse/check/ClickhouseChecker.kt

```diff
@@ -23,6 +25,8 @@ class ClickhouseChecker(
     @VisibleForTesting val tableName = "_airbyte_check_table_${clock.millis()}"
 
     override fun check(config: ClickhouseConfiguration) {
+        assert(!config.hostname.startsWith(PROTOCOL)) { PROTOCOL_ERR_MESSAGE }
```

Comment:
This would break with some unusal things such as `http+ssh://` unless you want to catch all those as well.